### PR TITLE
Refactor getter logic into test helpers

### PR DESCRIPTION
**Pull request by chancez**

### CODE REVIEW

File: test/testhelpers/helpers.go [hunk 1]
```diff
@@ -66,3 +67,66 @@ func NewPrestoTable(name, namespace string, columns []hive.Column) *v1alpha1.Pre
...
+		m[dataSource.Namespace+"/"+dataSource.Name] = dataSource
+	}
+	return &ReportDataSourceStore{m}
+}
+
+func (store *ReportDataSourceStore) GetReportDataSource(namespace, name string) (*v1alpha1.ReportDataSource, error) {
+	dataSource, ok := store.datasources[namespace+"/"+name]
+	if ok {
+		return dataSource, nil
+	} else {
```
Comment (timflannagan1) [inline test/testhelpers/helpers.go]: Can drop this `else` clause, and the other ones on line #'s 108 & 129

Comment (chancez) [reply test/testhelpers/helpers.go]: Good point.

**Pull request merged**